Firestore/Source/API/FIRFirestore+Internal.h

```diff
@@ -69,26 +68,6 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (const std::shared_ptr<util::AsyncQueue> &)workerQueue;
 
-/**
- * Attaches a listener for a snapshots-in-sync event. Server-generated
- * updates and local changes can affect multiple snapshot listeners.
- * The snapshots-in-sync event indicates that all listeners affected by
- * a given change have fired.
- *
- * NOTE: The snapshots-in-sync event only indicates that listeners are
- * in sync with each other, but does not relate to whether those
- * snapshots are in sync with the server. Use SnapshotMetadata in the
- * individual listeners to determine if a snapshot is from the cache or
- * the server.
- *
- * @param listener A callback to be called every time all snapshot
- * listeners are in sync with each other.
- * @return A FIRListenerRegistration object that can be used to remove the
- * listener.
- */
-- (id<FIRListenerRegistration>)addSnapshotsInSyncListener:(void (^)(void))listener
-    NS_SWIFT_NAME(addSnapshotsInSyncListener(_:));
-
```

Comment:
If you didn't already you might want to check if our tests that were using this now have an unnecessary import for `FIRFirestore+Internal.h`...  but not really a big deal.